From baac7c3d81801eaa87441f35e9e41530a5695a88 Mon Sep 17 00:00:00 2001
From: Craig Tatlor <ctatlor97@gmail.com>
Date: Mon, 21 May 2018 17:10:35 +0100
Subject: [PATCH 2/3] Build libc2dcolorconvert with BOARD_VNDK_VERSION

Includes needed shared libraries and adds includes that were missing.

Change-Id: I6cc41e3ff2f7a71329b893ee7f55b9eb57a0e3b6
---
 libc2dcolorconvert/Android.mk            | 2 +-
 libc2dcolorconvert/C2DColorConverter.cpp | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/libc2dcolorconvert/Android.mk b/libc2dcolorconvert/Android.mk
index 23239c4f2..b6610989e 100644
--- a/libc2dcolorconvert/Android.mk
+++ b/libc2dcolorconvert/Android.mk
@@ -13,7 +13,7 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_C_INCLUDES += $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include
 
-LOCAL_SHARED_LIBRARIES := liblog libdl
+LOCAL_SHARED_LIBRARIES := libutils liblog libdl
 
 LOCAL_MODULE_TAGS := optional
 
diff --git a/libc2dcolorconvert/C2DColorConverter.cpp b/libc2dcolorconvert/C2DColorConverter.cpp
index 04ddeb3ba..ce875bbf0 100644
--- a/libc2dcolorconvert/C2DColorConverter.cpp
+++ b/libc2dcolorconvert/C2DColorConverter.cpp
@@ -30,6 +30,7 @@
 #include <C2DColorConverter.h>
 #include <arm_neon.h>
 #include <stdlib.h>
+#include <unistd.h>
 #include <fcntl.h>
 #include <linux/msm_kgsl.h>
 #include <sys/ioctl.h>
-- 
2.17.0

